### PR TITLE
Task2 Scraping from website and saving to a file format

### PR DESCRIPTION
**Pull request by upexya**
This branch includes:
- Extracting data from a website
- Finding total number of pages and looping to get all records
- Storing data in **csv** and **json** format
- Storing data in csv file to **postgres** and **sqlite3**

Task remaining:
- Storing data in xml and yaml format
- Adding unit test data

### CODE REVIEW

File: Python/UpeachyaKansakar/tasks/task2/scraping.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import time
+import requests
+import urllib.request
+from pandas import DataFrame 
+from bs4 import BeautifulSoup
+
+URL = 'https://www.opencodez.com/category/web-development'
+
+def scrap_page_title(soupObject):
```
Comment (Basanta78) [inline Python/UpeachyaKansakar/tasks/task2/scraping.py]:
```suggestion
def scrap_page_title(soup_object):
```
Please follow snake case while naming all the variable name

File: Python/UpeachyaKansakar/tasks/task2/scraping.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        soup = BeautifulSoup(response.text, 'html.parser') 
+        all_titles = all_titles + scrap_page_title(soup)
+        all_authors = all_authors + scrap_page_authors(soup)
+        time.sleep(3)
+
+    save_as_csv(all_titles, all_authors)
+    save_as_json(all_titles, all_authors)
+
+    import pdb
+    pdb.set_trace()
```
Comment (Basanta78) [inline Python/UpeachyaKansakar/tasks/task2/scraping.py]: Please remove debugging codes

File: Python/UpeachyaKansakar/tasks/task2/storeToPostgres.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    conn = psycopg2.connect("host=localhost dbname=python_training user=postgresuser password=root")
+    cur = conn.cursor()
+
+    # NOTE: Query for creating table
+    cur.execute("""
+        CREATE TABLE articles(
+        title text,
+        author text
+    )
+    """)
```
Comment (Basanta78) [inline Python/UpeachyaKansakar/tasks/task2/storeToPostgres.py:14]: Try to use context manager 
with connection.cursor() as cursor:
            cursor.execute(sql query)

File: Python/UpeachyaKansakar/tasks/task2/storeToSqlite.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    connect = sqlite3.connect(database)
+    cursor = connect.cursor()
+
+    # Creating a new table
+    cursor.execute("""
+        CREATE TABLE articles(
+        title text,
+        author text
+    )
+    """)
```
Comment (Basanta78) [inline Python/UpeachyaKansakar/tasks/task2/storeToSqlite.py:19]: You can use context manager here
with con:
con.execute(
Reference:
https://docs.python.org/3/library/sqlite3.html#using-the-connection-as-a-context-manager

File: Python/UpeachyaKansakar/tasks/task2/scraping.py (hunk 1 above)
Comment (upexya) [reply Python/UpeachyaKansakar/tasks/task2/scraping.py]: :ok:

File: Python/UpeachyaKansakar/tasks/task2/scraping.py (hunk 2 above)
Comment (upexya) [reply Python/UpeachyaKansakar/tasks/task2/scraping.py]: :ok:

**Pull request merged**